src/radical/pilot/configs/resource_ornl.json

```diff
@@ -318,7 +318,7 @@
         "agent_config"                : "default",
         "agent_scheduler"             : "CONTINUOUS",
         "agent_spawner"               : "POPEN",
-        "agent_launch_method"         : "PRTE",
+        "agent_launch_method"         : "JSRUN",
```

Comment:
Is this intentional? I was confused by this being a PRTE PR.

Reply:
Ah, good catch - but yes, this is intentional.  The `agent_launch_method` is used to place sub-agents onto compute nodes, and jsrun is a good choice for doing so, in case we use agent configurations where the PRTE DVM and the HNP also runs on a compute node - in that case, we cannot use prun to place the sub-agent there, as the DVM would not be alive at that point.